src/main/java/zone/rong/loliasm/config/LoliConfig.java

```diff
@@ -71,7 +71,7 @@ public boolean shouldSkipClass(Class<?> clazz) {
     public boolean optimizeSomeRendering, stripUnnecessaryLocalsInRenderHelper;
     public boolean quickerEnableUniversalBucketCheck, stripInstancedRandomFromSoundEventAccessor, classCaching, copyScreenshotToClipboard, releaseScreenshotCache;
     public boolean fixBlockIEBaseArrayIndexOutOfBoundsException, cleanupChickenASMClassHierarchyManager, optimizeAmuletRelatedFunctions, labelCanonicalization, skipCraftTweakerRecalculatingSearchTrees, bwmBlastingOilOptimization, optimizeQMDBeamRenderer, repairEvilCraftEIOCompat, optimizeArcaneLockRendering, fixXU2CrafterCrash, disableXU2CrafterRendering;
-    public boolean fixAmuletHolderCapability, delayItemStackCapabilityInit;
+    public boolean fixAmuletHolderCapability, delayItemStackCapabilityInit, fixDrawersItemHandlers;
```

Comment:
Move this to after `disableXU2CrafterRendering` instead as its in the `modfixes` category